cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -980,11 +983,18 @@ def test_tissue_ontology_term_id_cell_culture_3(self, validator_with_adata, term
         obs.loc[obs.index[0], "tissue_type"] = "cell culture"
         obs.loc[obs.index[0], "tissue_ontology_term_id"] = term
         validator.validate_adata()
+
+        # Forbidden columns may be marked as either "not allowed" or "deprecated"
         assert validator.errors == [
             f"ERROR: '{term}' in 'tissue_ontology_term_id' is not allowed. When 'tissue_type' is "
             f"'cell culture', 'tissue_ontology_term_id' MUST be either a CL term "
             "(excluding 'CL:0000255' (eukaryotic cell), 'CL:0000257' (Eumycetozoan cell), "
             "and 'CL:0000548' (animal cell)) or 'unknown'."
+        ] or validator.errors == [
```

Comment:
this is basically the same issue as above with "not allowed" vs "deprecated" but with `tissue_ontology_term_id`